rholang-proto-build/src/main/scala/coop/rchain/rholang/build/RholangProtoBuild.scala

```diff
@@ -30,17 +30,22 @@ object RholangProtoBuild {
       case Right(term) => term.writeTo(new FileOutputStream(output.toFile))
     }
 
-  def createScalaArtifact(pkg: Option[String], name: String, proto: Path): String = {
+  def escapeSourceCode(source: Path): String =
+    Source.fromFile(source.toFile).getLines().mkString("\"\"\"\n", "\n|", "\"\"\".stripMargin")
```

Comment:
I think the `\n` in the first argument to `mkString` is not needed.
```
scala> def test(source: String): String = Source.fromFile(source).getLines().mkString("\"\"\"", "\n|", "\"\"\".stripMargin")
test: (source: String)String

scala>  val x = test("./rholang/examples/tut-hello.rho")
x: String =
"""new helloWorld, stdout(`rho:io:stdout`), stdoutAck(`rho:io:stdoutAck`) in {
|  contract helloWorld(@name) = {
|    new ack in {
|      stdoutAck!("Hello, ", *ack) |
|      for (_ <- ack) {
|        stdoutAck!(name, *ack) |
|        for (_ <- ack) {
|          stdout!("\n")
|        }
|      }
|    }
|  } |
|  helloWorld!("Joe")
|}""".stripMargin
```

Reply:
Done. @birchmd  Could you please help me merge this PR?

Reply:
Do you still not have permissions to merge? If not then I will merge on your behalf after the merge conflicts have been resolved.